pom.xml

```diff
@@ -85,6 +85,11 @@
 			<groupId>org.apache.httpcomponents.client5</groupId>
 			<artifactId>httpclient5</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>jcifs</groupId>
+			<artifactId>jcifs</artifactId>
+			<version>1.3.17</version>
```

Comment:
Use same convention as for the other version-parameters (for example `<version>${failsafe.version}</version>`)?